kserve.yaml

```diff
@@ -36,6 +37,7 @@ subpackages:
           modroot: .
           packages: ./cmd/${{range.key}}
           output: ${{range.key}}
+      - uses: strip
```

Comment:
Definitely a nit, but I think we handle stripping with ldflags we pass in the `go/build` pipeline (`-w`)

Reply:
yeah. on intell it got wheels everywhere.

Reply:
fixed.